### PR TITLE
Add BE locale

### PR DESCRIPTION
**Pull request by mawi12345**
Add BE locale (#703)

### CODE REVIEW

**Comment by mrenty** [comment]
I was a little confused by this PR since `BE` is the land code of Belgium. I think you mean `BY` for Belarus.

**Comment by leshakoss** [comment]
@mrenty ISO 639-1 code for belorusian is `be`. We use language codes, not country codes

**Comment by kossnocorp** [comment]
Same as with Serbian: I'm going to release it soon.

File: src/locale/be/build_distance_in_words_locale/index.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+function buildDistanceInWordsLocale () {
+  var distanceInWordsLocale = {
+    lessThanXSeconds: {
+      one: 'менш секунды',
```
[the remark this answers is not in the document]
Comment (kossnocorp) [reply src/locale/be/build_distance_in_words_locale/index.js]: @SpadarShut thanks, is there's anything else to improve?

[the remark this answers is not in the document]
Comment (SpadarShut) [reply src/locale/be/build_distance_in_words_locale/index.js]: Is there a way to have multiple plurals, not just one-many?

[the remark this answers is not in the document]
Comment (kossnocorp) [reply src/locale/be/build_distance_in_words_locale/index.js]: @SpadarShut it is possible: https://github.com/date-fns/date-fns/pull/716/files/4b5619551c7f5022308866df1d451527a64dac16#diff-fbd2607196d9849913aeee31b043270bR79

[the remark this answers is not in the document]
Comment (kossnocorp) [reply src/locale/be/build_distance_in_words_locale/index.js]: I appreciate the effort, but I can't work with incomplete suggestions. Also regarding "менш за секунду", Google shows more results for "менш секунды":

image 2018-12-10 at 7 08 39 pm

image 2018-12-10 at 7 08 22 pm

If you think it's wrong, please open PR. I can't wait for your answer, so I'm going to release as is.

**Comment by SpadarShut** [comment]
That's ok, this localization requires proper plurals which is a fair bit of
work and requires a PR anyway.

**Comment by kossnocorp** [comment]
The locale was released with `v1.30.1`. See: https://blog.date-fns.org/post/v130-is-out-m35nqh4m13r7

**Pull request merged**